### DISCUSSION TITLE
Clear Cache Buttons

### DISCUSSION THREAD

**Discussion by jeromeatneotek** [givanz/Vvveb#92] [General]
All the clear cache buttons have target _blank which means they cause a new tab to opened. I would prefer this to not happen,perhaps there is a reason for this?

**Comment by givanz** [comment]
This is done to prevent leaving the current page and is useful if you have unsaved changes.

The expected behavior for clear cache is not to leave the current page and it will be implemented with ajax, open in new page is just a temporary workaround.

**Reply by jeromeatneotek** [reply]
OK, thanks for clarifying. Do you have an ETA for the production ready solution?

**Comment by givanz** [comment]
I don't have an ETA I can only provide the order of the next steps.

Currently development is focused on the SEO plugin, after it's finished more testing will be done to check changes introduced to plugin system then small fixes and improvements like the ajax clear cache.

Expanding and improving documentation. 

Full testing and fixing bugs.

Version 1.0 release.  

Custom fields and more options for custom posts/products.

Custom fields will be built in steps with basic fields in the first stage and expanded in the following releases.

More payment options like paypal.

More components for the page builder and sections for the default theme.

Theme import with sample data and configuration for a fully working demo site.  

More themes and plugins and marketplace development with author section for designers and developers to submit their own extensions.

Faceted search with advanced filtering for products.

Subscriptions with recurring payment integration for stripe and paypal.

**Reply by jeromeatneotek** [reply]
Fair enough, sounds good. Happy to assist if your need any input.

**Reply by givanz** [reply]
Made some testing on pgsql and sqlite and fixed all bugs that I could find, let me know if you find anything that is still not working on pgsql.

Thanks.
